src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/Section/Field/FieldInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section\Field;
+
+/**
+ * @author Edouard Lescot <edouard.lescot@gmail.com>
+ * @author Oskar Stark <oskarstark@googlemail.com>
+ */
+interface FieldInterface
```

Comment:
Same here, there is no actual logic based on this interface, we should save us from maintaining an extra interface for the sake of it

Reply:
Agreed and removed 👍